tests/NamespacePrivateLinterTest.hack

```diff
@@ -20,7 +20,7 @@ final class NamespacePrivateLinterTest extends TestCase {
   <<__Override>>
   public function getCleanExamples(): vec<(string)> {
     return vec[
-      tuple('<?hh function test();'),
+      tuple('<?hh function test() {}'),
```

Comment:
This needs to be kept for docs.hhvm.com to be able to analyze builtins - this is sort-of valid when `<<__Native>>` is present

Reply:
Oh, using `;` as the body of a function was banned. There are no code examples left that use it. The codegen changed its type. Maybe a manual syntax example would restore the codegen to `CompoundStatement | SemiColon`?

Reply:
Yep, I'm adding a manual example which should result in that

It's banned in 'real' hack code, but `<<__Native>>` is an exception, and `<<__Native>>` is only valid in code that is built in to HHVM or an extension